.github/workflows/changelog.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  push:
+    branches: [ staging, trying, master ]
+  pull_request:
```

Comment:
Note that you need this to make it work for PRs from forks: https://github.com/stm32-rs/stm32f4xx-hal/blob/master/.github/workflows/changelog.yml#L1-L2

Reply:
Ah, I see. Then I can enable the detailed check version instead of the simple one, thanks!